scm/src/gmtb_scm_type_defs.f90

```diff
@@ -1477,6 +1565,9 @@ subroutine scm_state_create(scm_state, n_columns, n_levels, n_time_levels)
       scm_state%v_g(n_columns, n_levels), scm_state%dT_dt_rad(n_columns, n_levels), scm_state%h_advec_thil(n_columns, n_levels), &
       scm_state%h_advec_qt(n_columns, n_levels), scm_state%v_advec_thil(n_columns, n_levels), &
       scm_state%v_advec_qt(n_columns, n_levels), scm_state%pres_surf(n_columns,1), scm_state%T_surf(n_columns,1), &
+      scm_state%alvsf(n_columns), scm_state%alnsf(n_columns), scm_state%alvwf(n_columns), scm_state%alnwf(n_columns), &
+      scm_state%facsf(n_columns), scm_state%facwf(n_columns), &
+      scm_state%hprime(n_columns,14), scm_state%stddev(n_columns), &
```

Comment:
Note to self: investigate whether the hard-coded value of 14 can be replaced by "mntvar" variable (or whatever is used to dimension this array in the physics).